server/utils/urlParameterEncryption.ts

```diff
@@ -1,7 +1,8 @@
 import { enc, AES } from 'crypto-js'
```

Comment:
Would be good to move to the Node.js built-in crypto in the future as crypto-js is no longer maintained.